config/database.yml

```diff
@@ -41,6 +41,10 @@ sandbox:
   <<: *default
   url: <%= ENV["DATABASE_URL"] %>
 
+separation:
+  <<: *default
+  database: "early_careers_framework_separation"
```

Comment:
Is this required? The url contains the database created by terraform

Reply:
Ah no that should be `url: <%= ENV["DATABASE_URL"] %>` I'll update 👍